platform/util/platform-util-unavailableurls/pom.xml

```diff
@@ -24,6 +26,11 @@
     <packaging>jar</packaging>
     <dependencies>
         <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```

Comment:
OWASP is clean on the production code. It wants the Test class to use accessors in an inner class, but I don't feel it merits a code change.

```
/Users/aaronhoffer/repos/connexta/ddf/ddf/platform/util/platform-util-unavailableurls/src/test/java/org/codice/ddf/platform/util/http/UnavailableUrlsTest.java:203:31: warning: Variable 'runnable' must be private and have accessor methods.
/Users/aaronhoffer/repos/connexta/ddf/ddf/platform/util/platform-util-unavailableurls/src/test/java/org/codice/ddf/platform/util/http/UnavailableUrlsTest.java:205:27: warning: Variable 'timeout' must be private and have accessor methods.
/Users/aaronhoffer/repos/connexta/ddf/ddf/platform/util/platform-util-unavailableurls/src/test/java/org/codice/ddf/platform/util/http/UnavailableUrlsTest.java:207:31: warning: Variable 'unit' must be private and have accessor methods.
```